snakes_cafe.py

```diff
@@ -21,49 +21,65 @@
 ** To remove an item from your order, type the "remove <item name>" **
 ** To see your current order, type "order"                          **
 ** To quit at any time, type "quit"                                 **
+** To order from togo menu, type "togo"                             **
 **********************************************************************
 \n'''
 
 menu = {
     'Appetizers': {
-        'Wings': [0, 2.00],
-        'Calamari': [0, 2.00],
-        'Spring Rolls': [0, 2.00],
-        'Nachos': [0, 2.00],
-        'Spinach Dip': [0, 2.00],
-        'Sampler': [0, 2.00]
+        'Wings': [0, 2.00, 10],
```

Comment:
Since the values in this list are not related to each other it might be better to use a dictionary instead. For example:
```python
'Wings': {
             'quantity': 0,
             'price': 2.00,
             'stock': 10
         }
```